src/qt/anybutton.cpp

```diff
@@ -24,18 +25,87 @@ class wxQtPushButton : public wxQtEventSignalHandler< QPushButton, wxAnyButton >
 {
 
 public:
+    virtual bool eventFilter(QObject *watched, QEvent *event);
     wxQtPushButton( wxWindow *parent, wxAnyButton *handler);
 
 private:
     void clicked(bool);
+    wxAnyButton *m_button;
 };
 
 wxQtPushButton::wxQtPushButton(wxWindow *parent, wxAnyButton *handler)
     : wxQtEventSignalHandler< QPushButton, wxAnyButton >( parent, handler )
 {
+    m_button = handler;
+    installEventFilter( this );
     connect(this, &QPushButton::clicked, this, &wxQtPushButton::clicked);
 }
 
+bool wxQtPushButton::eventFilter(QObject *watched, QEvent *event)
+{
+// TODO: Disabled control shouldn't get any events
```

Comment:
It looks like when using eventFilter() all events are coming to control independentloy whether the QWidget is disabled or not.
I can update the PR putting the appropriate comment with an explanation.

Let me know.